drivers/volume/kdmp/kdmp.go

```diff
@@ -511,23 +549,23 @@ func (k *kdmp) StartRestore(
 
 		// create kdmp cr
 		dataExport := &kdmpapi.DataExport{}
+		// Adding required label for debugging
 		labels := make(map[string]string)
-		if len(restore.Name) <= labelNamelimit {
-			labels[restoreCRNameKey] = restore.Name
-		} else {
-			crLabelname := truncate.Truncate(restore.Name, labelNamelimit, "", truncate.PositionEnd)
-			labels[restoreCRNameKey] = strings.Trim(crLabelname, "-")
-		}
-		if len(bkpvInfo.PersistentVolumeClaim) <= labelNamelimit {
-			labels[pvcNameKey] = bkpvInfo.PersistentVolumeClaim
-		} else {
-			crLabelName := truncate.Truncate(bkpvInfo.PersistentVolumeClaim, labelNamelimit, "", truncate.PositionEnd)
-			labels[pvcNameKey] = strings.Trim(crLabelName, "-")
+		labels[applicationRestoreCRNameKey] = getValidLabel(restore.Name)
+		labels[applicationRestoreCRUIDKey] = getValidLabel(string(restore.UID))
```

Comment:
Can we add similar kinds of labels for `VolumeBackupCR` would be helpful when debugging for filteration?

Reply:
Fixed